content/manager/security.md

```diff
@@ -4,436 +4,701 @@ title: Security
+authentication by username and password. Other authentication methods (e.g. tokens) are currently not supported.
 
-{{% gsNote title="Note" %}}
-This client only supports authentication with username-password
-{{% /gsNote %}}
-
-## Other REST clients
-other REST clients (e.g. cURL) must explicitly add credentials to each request.<br>
+<br>
+### Other REST Clients
+other REST clients (e.g. cURL) must explicitly add the required header to each request.<br>
 For example:
 
- - Get the server status, authenticate with username and password:<br>
-curl -u 'MY_USERNAME':'MY_PASSWORD' MANAGER_IP/status
+ - Get the server's status, authenticate with username and password:<br>
+{{< gsHighlight  bash >}} curl -u 'MY_USERNAME':'MY_PASSWORD' <manager-ip-address>:<port>/api/v2/status{{< /gsHighlight >}}
  - Get a token, authenticate with username and password:<br>
-curl -u 'MY_USERNAME':'MY_PASSWORD' MANAGER_IP/tokens
+{{< gsHighlight  bash >}} curl -u 'MY_USERNAME':'MY_PASSWORD' <manager-ip-address>:<port>/api/v2/tokens {{< /gsHighlight >}}
  - Get all the blueprints, authenticate with a token:<br>
-curl -H 'Authentication-Token:MY_TOKEN' MANAGER_IP/blueprints
-
+{{< gsHighlight  bash >}} curl -H 'Authentication-Token:MY_TOKEN' <manager-ip-address>:<port>/api/v2/blueprints {{< /gsHighlight >}}
 
-# Behind the Scenes / Advanced
+<br>
+# Advanced
+## Internal Communication
 
-## Secured Request Flow
+Currently, communication between Cloudify agents and the manager or within the manager itself does not go through
+authentication or authorization. <br>
+Instead, these requests are sent over port 8101, used by the REST service to identify them as internal and allow them to
+proceed directly to the requested endpoint.
 
-![request-flow]({{< img "guide/request_flow.png" >}})
-
-## Internal communication between the Cloudify manager and other Cloudify components
-
-Currently, communication between the Cloudify agents and the Cloudify manager does not go through authentication - instead, REST calls from the agents to the manager are done to port 8101, which has the same general behavior as port 80, yet the REST service lets requests made to this port through without having them go through any of the security mechanisms.
-
-The usage of this port, however, is restricted to components on the same subnet as the Cloudify manager alone. This is done using specific security group rules, which are set up during bootstrap (for example, see [this rule in the Openstack manager blueprint](https://github.com/cloudify-cosmo/cloudify-manager-blueprints/blob/master/openstack/openstack-manager-blueprint.yaml#L206)). It is therefore impossible to bypass the security mechanisms from outside the manager's internal network by making REST calls directly to the 8101 port.
+The usage of this port, however, must be restricted to components on the same subnet as the Cloudify manager alone.
```

Comment:
Not sure subnet is the term we want to use here, but rather, the user should make sure that this port is reachable only by internal components. subnet will be a possible way to implement this requirement